mods/doors/init.lua

```diff
@@ -636,7 +646,15 @@ function doors.register_trapdoor(name, def)
 
 	def_closed.node_box = {
 		type = "fixed",
-		fixed = {-0.5, -0.5, -0.5, 0.5, -6/16, 0.5}
+		fixed = {
+                    {-0.5, -0.5, -0.5, 0.5, -6/16, -5/16},
+                    {-0.5, -0.5, 5/16, 0.5, -6/16, 0.5},
+                    {-0.5, -0.5, -5/16, -5/16, -6/16, 5/16},
+                    {5/16, -0.5, -5/16, 0.5, -6/16, 5/16},
+                    {-5/16, -0.5, -2/16, 5/16, -6/16, 2/16},
+                    {-2/16, -0.5, -5/16, 2/16, -6/16, 2/16},
+                    {-2/16, -0.5, 2/16, 2/16, -6/16, 6/16}
```

Comment:
You don't need so many boxes. Boxes can overlap.
Combine the two last boxes to:
```lua
{-2/16, -0.5, -5/16, 2/16, -6/16, 6/16}
```
As said, the boxes can overlap, so you do not have to care so much and can simply do something like this:
```lua
{-0.5, -0.5, -0.5, -5/16, -3/8, 0.5},
{-1/8, -0.5, -0.5, 1/8, -3/8, 0.5},
{5/16, -0.5, -0.5, 0.5, -3/8, 0.5},
{-0.5, -0.5, -0.5, 0.5, -3/8, -5/16},
{-0.5, -0.5, -1/8, 0.5, -3/8, 1/8},
{-0.5, -0.5, 5/16, 0.5, -3/8, 0.5}
```

(I guess using only 4 boxes where 3 of them are for the holes with `backface_culling = false` would be less efficient as `4*2*6 = 8*6 > 6*6`.)

Reply:
OK, I'll remember that. Thanks for the tip.

Reply:
But if I remove that extra box, this is what happens:
![test](https://user-images.githubusercontent.com/49789044/58519578-33819880-81ac-11e9-9541-a8f6f461fad7.png)
Removd ---> Not removed

Reply:
Done! I did that for both trapdoor models.

Reply:
I agree only 6 boxes are needed.